config/migrations/2024/20241017085951-fix-bindingStart-dates.sparql

```diff
@@ -0,0 +1,23 @@
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX lmb: <http://lblod.data.gift/vocabularies/lmb/>
+PREFIX bestuursperiode: <http://data.lblod.info/id/concept/Bestuursperiode/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+DELETE {
+  GRAPH ?g {
+    ?orgaan mandaat:bindingStart ?start.
+  }
+}
+INSERT {
+  GRAPH ?g {
+    ?orgaan mandaat:bindingStart "2024-12-04T00:00:00"^^xsd:dateTime .
+  }
+}
+WHERE {
+  GRAPH ?g {
+    ?orgaan lmb:heeftBestuursperiode bestuursperiode:96efb929-5d83-48fa-bfbb-b98dfb1180c7.
+    OPTIONAL {
+      ?orgaan mandaat:bindingStart ?start.
+    }
+  }
+};
```

Comment:
@bfidlers  As your PR was approved I assume it works in the migrations service. But the semicolon should not be here :D

Reply:
huh weird, I gues the migration service thought, don't care query already ended